src/DevBetterWeb.Core/Entities/Member.cs

```diff
@@ -123,11 +126,14 @@ public void UpdateName(string? firstName, string? lastName, bool isEvent = true)
 		}
 	}
 
-	public void UpdateAddress(string? address, bool isEvent = true)
+	public void UpdateAddress(string? address, string? city, string? country, string? postalCode, bool isEvent = true)
 	{
 		if (Address == address) return;
 
 		Address = address;
+		City = city;
+		Country = country;
+		PostalCode = postalCode;
 
```

Comment:
Do we need similar equality checks for the new fields like in `address`?

Reply:
This checks will not work in the same way need to be inverted.